ntbs-service/Services/AuditService.cs

```diff
@@ -32,5 +34,18 @@ public async Task AuditNotificationReadAsync(int notificationId, NotificationAud
                 RootEntities.Notification,
                 notificationIdString);
         }
+
+        public async Task AuditUnmatchSpecimen(int notificationId, string labReferenceNumber, string userName)
+        {
+            await _auditContext.AuditOperationAsync(
+                labReferenceNumber,
+                RootEntities.Specimen,

```

Comment:
Not sure if this wants to be in the `RootEntites` since, we aren't using it as a root entity - @conorsheehansw might have some intuition about how to best structure it..?

Reply:
I might be inclined towards:
```
labReferenceNumber,
typeof(MatchedSpecimen).Name,
null,
UNMATCH_EVENT,
userName,
RootEntites.Notification,
notificationId.ToString()
```

Though this would be on the back of my changeset which is currently WIP review.

Reply:
I have made the changes. Pushing to master now